.github/workflows/connect-dev-release-test.yml

```diff
@@ -80,7 +80,8 @@ jobs:
     with:
       test-name: passphrase.test
       DEV_SERVER_HOSTNAME: dev.suite.sldev.cz
-      run-webextension: true
+      # currently disable due to instability
+      # run-webextension: true
```

Comment:
Why don't we run it only on nightly runs instead of totally disabling it?

Reply:
Ok I think that's a reasonable compromise, but in the end we want to fix the cause of the flakiness

This should do it, right?
```
${{ github.event_name == 'schedule' }}
```

Reply:
Of course, we need to fix the cause of flakiness but I would feel safer if we see those test running at least once a day.

Yes, I think `${{ github.event_name == 'schedule' }}` should work as expected there.